templates/archive.html

```diff
@@ -4,37 +4,10 @@
 </div>
 <!-- /wp:group -->
 
-<!-- wp:query {"tagName":"main","layout":{"type":"constrained"}} -->
-<main class="wp-block-query">
-	<!-- wp:query-title {"type":"archive","style":{"spacing":{"margin":{"bottom":"100px"}}}} /-->
-
-	<!-- wp:post-template -->
-		<!-- wp:group {"tagName":"article"} -->
-		<article class="wp-block-group">
-			<!-- wp:post-title {"isLink":true} /-->
-			<!-- wp:post-featured-image {"isLink":true} /-->
-
-			<!-- wp:post-excerpt /-->
-			<!-- wp:template-part {"slug":"post-meta"} /-->
-
-			<!-- wp:spacer {"height":40} -->
-			<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
-			<!-- /wp:spacer -->
-		</article>
-		<!-- /wp:group -->
-
-	<!-- /wp:post-template -->
-
-	<!-- wp:group {"layout":{"type":"constrained"}} -->
-	<div class="wp-block-group">
-		<!-- wp:query-pagination -->
-			<!-- wp:query-pagination-previous /-->
-			<!-- wp:query-pagination-numbers /-->
-			<!-- wp:query-pagination-next /-->
-		<!-- /wp:query-pagination -->
-	</div>
-	<!-- /wp:group -->
+<!-- wp:group {"tagName":"main","style":{"spacing":{"blockGap":"0","margin":{"top":"0"}}},"layout":{"type":"default"}} -->
+<main class="wp-block-group" style="margin-top:0">
+	<!-- wp:pattern {"slug":"twentytwentyfour/archive"} /-->
```

Comment:
Since the only difference between archive and index is the existence of the query title, can we have this be index.html instead? The query title shouldn't show for the index template (and the index template that we have right now in the repo is incorrect if we look at the design)

Reply:
do we delete archive? it falls back to index.

Reply:
the index still needs an H1

Reply:
yeah the idea would be to delete archive. Looking at the design, I'm not sure what the h1 should be! What do you think?

Reply:
There is no way to conditionally and an H1 to index and not to the archives if archive.html is deleted.

Reply:
yeah, I know, but what would we add as h1 anyway?

Reply:
_Watch, Read, Listen_

Reply:
that makes sense. Can we movew the query title outside the query loop and just have a template for each that does heading + pattern for the query loop, then?

Reply:
Yeah

Reply:
https://github.com/WordPress/twentytwentyfour/pull/374